nodebuilder/header/flags.go

```diff
@@ -50,28 +47,24 @@ func TrustedPeersFlags() *flag.FlagSet {
 	return flags
 }
 
-// ParseTrustedPeerFlags parses Header package flags from the given cmd and applies values to Env.
+// ParseTrustedPeerFlags parses Header package flags from the given cmd and saves them to the passed config.
```

Comment:
same thing here, just leave it as `applies`

Reply:
done